### ISSUE TITLE
NSFW filter

### ISSUE THREAD

**Issue by arsaizdihar** [D3vd/Meme_Api#76]
Can we filter so that if the subreddit is NSFW it will return nothing?

**Comment by TheScientist101** [comment]
Yeah, this would be useful, but right now I am just requesting the meme and checking if NSFW is false, otherwise get another meme.

**Comment by JaffaCakes118** [comment]
Yes this is a must need, I'm worried that the bot might randomly get a meme which is NSFW in a non NSFW channel.

**Comment by Jvala2** [comment]
> Yeah, this would be useful, but right now I am just requesting the meme and checking if NSFW is false, otherwise get another meme.

Can I see the code you're using to do that?

**Comment by JaffaCakes118** [comment]
> 
> 
> > Yeah, this would be useful, but right now I am just requesting the meme and checking if NSFW is false, otherwise get another meme.
> 
> Can I see the code you're using to do that?

Here's how I'm doing it

		async function getMeme(url) {
			try {
				const response = await axios.get(url);
				if(response.data.nsfw) {
					return getMeme(url);
				}
				else{
					return response.data;
				}

			}
			catch (e) {
				return message.channel.send('An error has occured, try again!');
			}
		}

**Comment by TheScientist101** [comment]
> > Yeah, this would be useful, but right now I am just requesting the meme and checking if NSFW is false, otherwise get another meme.
> 
> 
> 
> Can I see the code you're using to do that?

There are no do while loops in python so I implemented it as below. I am using it for a discord bot so that is why it is ctx.send. The full code is in
